battery.cpp

```diff
@@ -32,3 +35,27 @@ void Battery::decrease_battery(int val) {
 int Battery::get_battery() { return battery; }
 
 int Battery::get_range() { return range; }
+
+void Battery::on() {
+  this->ui->lowBatteryWarning->setPixmap(lowBatteryIcon);
+}
+
+void Battery::off() {
+  this->ui->lowBatteryWarning->setText("OFF");
+}
+
+void Battery::CheckRange() {
+  bool isLow = false;
+
+  if (get_range() <= 30) {
```

Comment:
You can simplify the `if/else` statements here as such:
```suggestion
  if (get_range() <= 30) {
      on(LOW_BATTERY);
```